lib/inspec/fetcher.rb

```diff
@@ -38,9 +38,9 @@ def self.fetcher(version)
 end
 
 # TODO: remove. require up, not down.
-require "fetchers/local"
-require "fetchers/url"
-require "fetchers/git"
+require "inspec/fetchers/local"
+require "inspec/fetchers/url"
+require "inspec/fetchers/git"
```

Comment:
Since the namespace is `Inspec::Fetcher`, then maybe the directory should be renamed to singular?